--- a/patches/test.cpp	2021-03-18 15:56:46.899609711 -0400
+++ b/patches/test.cpp	2021-03-19 16:31:16.572950070 -0400
@@ -11,7 +11,7 @@ void copy_it_to_string( element_t &dst, con
     dst.length = strncpy( dst.buffer, src, dst.max_length );
     if( dst.length >= dst.max_length )
         dst.length = dst.max_length - 1;
-    dst.buffer[dst.length] = '\0';
+    dst.buffer[dst.length - 1] = '\0';
     return;
 }
 
