--- a/patches/test.cpp	2021-03-24 21:14:18.228875784 -0400
+++ b/patches/test.cpp	2021-03-29 16:18:19.136324229 -0400
@@ -17,12 +17,12 @@ void copy_to_string( element_t &dst, con
 
 std::string MakeString( const char * str ) {
     char buffer[256];
-    memset( localString, 0, 256 );
-    snprintf( localString, 256, "%s", str );
+    memset( buffer, 0, 256 );
+    snprintf( buffer, 256, "%s", str );
 
-    printf( "Location of the string: %p\n", localString );
+    printf( "Location of the string: %p\n", buffer );
 
-    return localString;
+    return buffer;
 }
 
 int main( int argc, char ** argv ) {
